### DISCUSSION TITLE
How do I know the VPN is working fine in my containers?

### DISCUSSION THREAD

**Discussion by JulenDuran** [qdm12/gluetun#2263] [Q&A] [answered]
Hello all, 

I do have Gluetun configurated with a NordVPN account and it seems to be working fine. The logs shows in [ip getter] field the VPN IP and no errors are shown. I do have some containers routed to he gluetun network.

There is no way to know if the containers routed to gluetun network are sending their traffic throught the VPN so I installed a Chrome container to perform a simple "Whats my IP" test. This container is also routed to the gluetun network... well, to my surprise the IP shown is my real IP no the VPN IP... so now I dont know it the VPN is working fine or not.

Any ideas?

Thank you all!!

**Comment by frepke** [comment]
How did you connect the Chrome container to the Gluetun container?

**Reply by JulenDuran** [reply]
Assigned the gluetun network to the container in the compose file.

```
networks:
  gluetun:
    name: gluetun
```

In Portainer can be seen the container asigned correctly to the network

Tks for the reply!

**Comment by frepke** [comment]
Can you try to add this to your compose file:
```
network_mode: "service:gluetun"
    depends_on:
      gluetun:
        condition: service_healthy

**Reply by frepke** [reply]
Or only change the `network_mode`

**Reply by JulenDuran** [reply, marked as answer]
I just found a guide to do it in Portainer directly.

This is the guide I found: https://dbtechreviews.com/2021/02/10/run-all-your-docker-containers-through-a-gluetun-vpn-container/

Now works like a charm

However frepke thanks a lot for your instructions!!
